mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/store/MantisStorageProvider.java

```diff
@@ -183,4 +186,12 @@ List<MantisWorkerMetadataWritable> getArchivedWorkers(String jobid)
     void deleteExpiredDisableTaskExecutorRequest(DisableTaskExecutorsRequest request) throws IOException;
 
     List<DisableTaskExecutorsRequest> loadAllDisableTaskExecutorsRequests(ClusterID clusterID) throws IOException;
+
+    void storeNewJobArtifact(JobArtifact jobArtifact) throws IOException;
+
+    boolean jobArtifactExists(ArtifactID artifactID);
+
+    List<String> searchJobArtifactProjects(@Nullable String prefix) throws IOException;
```

Comment:
Add a comment on what the return value is. I would assume that this returns the artifact name but make it clear.

Reply:
+1.